From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Mon, 29 Aug 2016 12:34:42 -0400
Subject: avoid regenerating defsincdate (use shipped file)

upstream ships doc/defsincdate in its tarballs.  but doc/Makefile.am
tries to rewrite doc/defsincdate if it notices that any of the files
have been modified more recently, and it does so assuming that we're
running from a git repo.

However, we'd rather ship the documents cleanly without regenerating
defsincdate -- we don't have a git repo available (debian builds from
upstream tarballs) and any changes to the texinfo files (e.g. from
debian/patches/) might result in different dates on the files than we
expect after they're applied by dpkg or quilt or whatever, which makes
the datestamp unreproducible.
---
 doc/Makefile.am | 9 ---------
 1 file changed, 9 deletions(-)

Patch-Source: https://sources.debian.org/data/main/g/gnupg2/2.2.27-2/debian/patches/debian-packaging/avoid-regenerating-defsincdate-use-shipped-file.patch

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 2b882c3..6be571b 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -178,15 +178,6 @@ $(myman_pages) gnupg.7 : yat2m-stamp defs.inc
 
 dist-hook: defsincdate
 
-defsincdate: $(gnupg_TEXINFOS)
-	: >defsincdate ; \
-	if test -e $(top_srcdir)/.git; then \
-	  (cd $(srcdir) && git log -1 --format='%ct' \
-               -- $(gnupg_TEXINFOS) 2>/dev/null) >>defsincdate; \
-        elif test x"$$SOURCE_DATE_EPOCH" != x; then   \
-	   echo "$$SOURCE_DATE_EPOCH" >>defsincdate ; \
-	fi
-
 defs.inc : defsincdate Makefile mkdefsinc
 	incd="`test -f defsincdate || echo '$(srcdir)/'`defsincdate"; \
 	./mkdefsinc -C $(srcdir) --date "`cat $$incd 2>/dev/null`" \
